qt/python/mantidqt/utils/qt/__init__.py

```diff
@@ -147,6 +151,10 @@ def create_action(parent, text, on_triggered=None, shortcut=None,
     if icon_name is not None:
         action.setIcon(get_icon(icon_name))
 
+    # shortcuts in context menus option is only available after Qt 5.10
+    if 5 == QT_VERSION_INT[0] and 10 <= QT_VERSION_INT[1] and shortcut_visible_in_context_menu:
```

Comment:
What about the inevitable Qt 6? Would a `hasattr` check on `action` be better?

Reply:
Good call!